users/views.py

```diff
@@ -1,3 +1,75 @@
-# from django.shortcuts import render
+from users.models import User
+from django.contrib import messages
+from users.forms import MyUserCreationForm
+from django.shortcuts import redirect, render
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth import authenticate, login, logout
 
-# Create your views here.
+
+def login_page(request):
+
+    if request.user.is_authenticated:
+        return redirect("home")
+
+    if request.method == "POST":
+        email = request.POST.get("email")
+        password = request.POST.get("password")
+
+        print(f"username: {email}\npassword: {password}")
+
+        try:
+            user = User.objects.get(email=email)
+        except Exception:
+            messages.error(request, "user does not exist")
+
+        user = authenticate(request, email=email, password=password)
+
+        if user is not None:
+            login(request, user)
+
+            return redirect("home")
+
+        else:
+            messages.error(request, "username or password does not exists")
+
+    form = MyUserCreationForm()
+
+    return render(request, "login/register_login.html", {'form': form, 'page': 'login'})
+
+
+def reqister_page(request):
```

Comment:
There if typo in the function name.

```suggestion
def register_page(request):
```

Reply:
done. thanks 😄